cvat/apps/documentation/user_guide.md

```diff
@@ -1,232 +1,442 @@
+The model manager allows you to manage your deep learning (DL) models uploaded for auto annotation.
+Using the functionality you can upload, update or delete a specific DL model.
+Use "Auto annotation" button to pre annotate a task using one of your DL models. [Read more](/apps/auto_annotation)
 
-    __Segment size__. Use this option to divide a huge dataset into a few smaller segments.
+![](static/documentation/images/image104.jpg)
 
-    __Image Quality__. Use this option to specify quality of uploaded images. The option makes it faster to load high-quality datasets. Use the value from ``1`` (completely compressed images) to ``95`` (almost not compressed images).
+### Search
 
-    Push ``Submit`` button and it will be added into the list of annotation tasks. Finally you should see something similar to the figure below:
+There are several options how to use the search.
+It may be task's name or id, owner's name, task's status, assignee's name and so on. The search is case insensitive.
 
-    ![](static/documentation/images/image006.jpg)
+![](static/documentation/images/image100.jpg)
 
-3. Follow a link inside ``Jobs`` section to start annotation process. In some cases you can have several links. It depends on size of your task and ``Overlap Size`` and ``Segment Size`` parameters. To improve UX only several first frames will be loaded and you will be able to annotate first images. Other frames will be loaded in background.
+## Interface of the annotation tool
 
-    ![](static/documentation/images/image007.jpg)
+The tool consists of:
+- ``Workspace`` — where images are shown;
+- ``Bottom panel`` (under workspace) — for navigation, filtering annotation and accessing tools' menu;
+- ``Side panel`` — contains two lists: objects (on the frame) and labels (of objects on the frame);
+- ``Bottom side panel`` — is a type of/creating/merging/grouping annotation;
+
+![](static/documentation/images/image034.jpg)
+
+There is also:
+- ``Settings`` (F2) — pop-up in the bottom panel, contains different parameters
+  which can be adjusted according to the user's needs
+- ``Context menu`` — available on right mouse button.
 
 ### Basic navigation
 
-1. Use arrows below to move on next/previous frame. Almost every button is covered by a shortcut. To get a hint about the shortcut just put your mouse pointer over an UI element.
+1.  Use arrows below to move on next/previous frame.
+    Use the scroll bar slider to scroll through frames.
+    Almost every button is covered by a shortcut.
+    To get a hint about a shortcut, just put your mouse pointer over an UI element.
 
     ![](static/documentation/images/image008.jpg)
 
-2. An image can be zoom in/out using mouse's wheel. The image will be zoomed relatively your current cursor position. Thus if you point on an object it will be under your mouse during zooming process.
+1.  An image can be zoom in/out using mouse's wheel. The image will be zoomed relatively your current cursor position.
```

Comment:
can be zoomed (but "scaled" sounds better)
just "current cursor position" without your